hid-xpadneo/dkms.post_install

```diff
@@ -3,7 +3,12 @@
 ERTM_OVERRIDE="/etc/modprobe.d/99-xpadneo-bluetooth.conf"
 DISABLE_ERTM="/sys/module/bluetooth/parameters/disable_ertm"
 
-if [ "$(readlink "${ERTM_OVERRIDE}" 2>/dev/null)" = "/dev/null" ]; then
+KERNEL_MAJOR=`uname -r | awk -F'.' '{print $1}'`
+KERNEL_MINOR=`uname -r | awk -F'.' '{print $2}'`
+
+if [ "$KERNEL_MAJOR" -ge 6 ] || ([ "$KERNEL_MAJOR" -ge 5 ] && [ "$KERNEL_MINOR" -ge 12 ]); then
```

Comment:
I'd probably use something like `uname -r | awk -F. '{ printf "%03d%03d",$1,$2 }'` and compare against `005012`.